StripeApplePay/StripeApplePay/Source/ApplePayContext/STPApplePayContext.swift

```diff
@@ -577,259 +579,261 @@ public class STPApplePayContext: NSObject, PKPaymentAuthorizationControllerDeleg
             }
         }
 
-        // 1. Create PaymentMethod
-        StripeAPI.PaymentMethod.create(apiClient: apiClient, payment: payment, clientAttributionMetadata: clientAttributionMetadata) { result in
-            guard let paymentMethod = try? result.get(), self.authorizationController != nil else {
-                if case .failure(let error) = result {
-                    let errorMessage = "Failed on token creation"
-                    let errorAnalytic = ErrorAnalytic(event: .unexpectedApplePayError,
-                                                      error: error,
-                                                      additionalNonPIIParams: [
-                                                        "error_message": errorMessage
-                                                      ])
-                    handleFinalState(.error, error, errorAnalytic)
-                } else {
-                    let errorMessage = "Failed while creating paymentMethod due to internal error"
-                    let errorAnalytic = ErrorAnalytic(event: .unexpectedApplePayError,
-                                                      error: InternalError.invalidState,
-                                                      additionalNonPIIParams: [
-                                                        "error_message": errorMessage
-                                                      ])
-
-                    self.analyticsClient.log(analytic: errorAnalytic, apiClient: self.apiClient)
-                    handleFinalState(.error, nil, errorAnalytic)
-                }
-                return
-            }
-
-            let paymentMethodCompletion: STPIntentClientSecretCompletionBlock = {
-                clientSecret,
-                intentCreationError in
-                guard let clientSecret = clientSecret, intentCreationError == nil,
-                    self.authorizationController != nil
-                else {
-                    let errorMessage = "Failed on payment method completion"
-                    let errorAnalytic = ErrorAnalytic(event: .unexpectedApplePayError,
-                                                      error: InternalError.invalidState,
-                                                      additionalNonPIIParams: [
-                                                        "error_message": errorMessage,
-                                                      ])
-                    handleFinalState(.error, intentCreationError, errorAnalytic)
+        PassiveCaptcha.fetchPassiveHCaptchaToken(passiveCaptcha: passiveCaptcha) { hcaptchaToken in
```

Comment:
Do we want to do this for Apple Pay? I think the sort of things hCaptcha prevents are less of a concern for Apple Pay transaction, so I'd be inclined to leave it out for latency reasons